Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -754,21 +754,34 @@ CGPathRef CGPathCreateCopyByStrokingPath(
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
```

Comment:
I think this should be interoperable.  It would be a caveat if ref plat created mutable copy and we didn't.